### PR TITLE
Add deprecation warnings to old header components

### PR DESCRIPTION
**Pull request by ohskylab**
TP: https://rockabox.tpondemand.com/entity/11504

### CODE REVIEW

File: src/rb-header/rb-header-directive.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ define([
      * @restrict E
      *
      * @description
+     * DEPRECATED. Please use rg-header, which encompases functionality from this component.
```
Comment (nwhite89) [inline src/rb-header/rb-header-directive.js]: could you use http://usejsdoc.org/tags-deprecated.html please

Comment (ohskylab) [inline src/rb-header/rb-header-directive.js]: Cool, didn't know about that. Presume just in the directives, styles and demos OK to stand as they are?

Comment (nwhite89) [inline src/rb-header/rb-header-directive.js]: exactly cheers merge on Squash :)

**Comment by ohskylab** [comment]
@nwhite89 autosquashed...

File: src/rb-page-header/rb-page-header-directive.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ define([
      * @restrict E
      *
      * @description
```
Comment (chrisvxd) [inline src/rb-page-header/rb-page-header-directive.js]: Missing `@deprecated`

**Comment by ohskylab** [comment]
@chrisvxd added (doh)

**Pull request merged**